static/js/subs.js

```diff
@@ -366,6 +374,10 @@ exports.mark_sub_unsubscribed = function (sub) {
             settings.collapse('hide');
         }
 
+        // Update subscribers count
+        var count = stream_data.get_subscribers_count(sub.name) - 1;
```

Comment:
@timabbott I agree with all your comments above. Can you explain how can I count the elements that have `.subscribed=true`, I didn't get you here.

Kartik Maji